### PR TITLE
DeveloperGuide.adoc: Add Use Cases 1 to 6

### LINKED ISSUE #74: Update first half of use case for Developer Guide

### PR DESCRIPTION
**Pull request by khsc96**

### CODE REVIEW

**Review by Exeexe93 — APPROVED** [review]
LGTM

**Review by JeremyLoh — APPROVED** [review]
LGTM

**Comment by jerrenss** [comment]
The use cases are well-defined and easy to follow. I think the team did a great job on that!

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -331,32 +331,146 @@ _{More to be added}_
...
 Use case ends.
 
-* 3a. The given index is invalid.
+[discrete]
+=== Use case: UC02 - Clear all orders
+
+*MSS*
+
+1.  User requests to clear all orders.
+2.  Delino clear all existing orders.
```
Comment (Yuanyu97) [inline docs/DeveloperGuide.adoc:359]: Maybe Delino can prompt the user to ask if the user really wants to delete all the orders before actually deleting it

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -331,32 +331,146 @@ _{More to be added}_
...
+** 1b1. Delino shows no order to be cleared message.
++
+Use case ends.
+
+[discrete]
+=== Use case: UC03 - Delete an order
+
+*MSS*
+
+1.  User requests to list orders (UC10).
```
Comment (Yuanyu97) [inline docs/DeveloperGuide.adoc:385]: Does this mean that the user has to key in the command to list the orders before the user can delete a specific order? If that is the case, maybe you guys can remove step one, and leave it up to the user whether he or she wants to see the list before deleting an order.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -331,32 +331,146 @@ _{More to be added}_
...
+2.  User requests to delete a specific order in the list.
+3.  Delino deletes the order.
```
Comment (Yuanyu97) [inline docs/DeveloperGuide.adoc:387]: Similar to my review on the deleting all the orders, maybe Delino can ask the user if he or she really wants to delete the orde before actually deleting the order.

**Review by Yuanyu97 — COMMENTED** [review]
Overall a good job! Keep up the good work.

**Pull request merged**